src/schema.rs

```diff
@@ -45,6 +45,13 @@ pub struct ModuleV1 {
     pub contracts: BTreeMap<String, ContractV1>,
 }
 
+/// Contains all the contract schemas for a module newer than V1 module.
+#[derive(Debug, Clone)]
+pub enum VersionedModule {
+    /// Versioned schema module version 0
```

Comment:
This is a confusing comment. I guess it has nothing to do with V0 contracts.

It's also very confusing that your constructor is V0, but the type inside it is called ModuleV1. Why not use `V1`?

Reply:
Well, my thought was that this is the first version of the _versioned_ schema module, but I see how that is needlessly confusing.
I'll change it

Reply:
Yeah, too many versions of different things at this point :)